README.md

```diff
@@ -34,19 +37,44 @@ deploy_xdcstore(
 )
 ```
 
-## Deploy with a one-liner
+## Deploy with few CLI commands
+
+You can also use this module
+to deploy xdcstore and xdcget
+with these few CLI commands:
 
-It can also be used to deploy xdcstore and xdcget with an ad-hoc command like this:
 ```
+# install pyinfra, and this module
+git clone https://github.com/deltachat/pyinfra-xdcstore
 pip install --user .
-XDCSTORE_EMAIL=xdcstore@example.org XDCSTORE_PASSWORD=p4ssw0rd pyinfra --ssh-user root -- example.org pyinfra_xdcstore.deploy_xdcstore
+
+# configure the bot's credentials
+export XDCSTORE_EMAIL=xdcstore@example.org
+export XDCSTORE_PASSWORD=p4ssw0rd
+
+# run the deployment
+pyinfra --ssh-user root -- <your_server> deploy.py
 ```
 
-Additional environment variables you can use:
+Additional environment variables you can (and should) use:
```

Comment:
Because you can use either github *or* codeberg, but technically don't need both of them, and if you don't provide `XDCGET_UNIX_USER` then a reasonable default is chosen/created. So the only variables which are really not optional are `XDCSTORE_EMAIL` and `XDCSTORE_PASSWORD`.